src/platform/ESP32/ConnectivityManagerImpl.cpp

```diff
@@ -30,6 +30,7 @@
 #include <support/logging/CHIPLogging.h>
 
 #include "esp_event.h"
+#include "esp_netif.h"
```

Comment:
So CI seems to be failing because this header can't be found. I think the Docker Images CI uses need to be updated.

Reply:
@sagar-apple That's right. `esp-netif.h` header is added in ESP-IDF release v4.1 and was not present in v4.0

Reply:
@rwalker-apple whats the right order to fix this in? Does he need to update the dockerfile in this PR and then publish the images?

Reply:
yeah, Dockerfile, Docker push, rebase :-(

a shorter circuit would be @dhrishi, can you take a crack at the esp32 image update?  if you add it to this PR, I can push to docker.io to get this to land...

Reply:
@rwalker-apple Sure, no worries